### DISCUSSION TITLE
Allow iframe pop up for 3DS authentication

### DISCUSSION THREAD

**Discussion by nfourtythree** [craftcms/commerce-stripe#170] [Ideas]

**Comment by ldennison-bsd** [comment]
I assume this feature request is referencing Stripe's native "pre-built modal" support for 3DS authentication as documented on their page about 3DS Authentication for Web: https://stripe.com/docs/payments/3d-secure/web

It would seem that if the JavaScript and related workflow is modified to use the handleCardPayment method, then Stripe will automatically handle 3DS authentication through a popup modal interface. This is much cleaner than the redirect workflow.

Is this part of the roadmap for Stripe for Commerce?

**Comment by samhibberd** [comment]
This is something we would love to see added, is anything planned on the roadmap yet?

**Comment by icreatestuff** [comment]
We'd love to see this implemented too.

**Comment by j-greig** [comment]
Ditto
